docs/source/algorithms/ReflectometryILLPreprocess-v1.rst

```diff
@@ -23,14 +25,26 @@ The algorithm can be thought as an 'advanced loader', and should be used to load
 
 The *OutputWorkspace* can be further fed to :ref:`ReflectometryILLSumForeground <algm-ReflectometryILLSumForeground>`.
 
+The algorithm adds the following sample log entries to the *OutputWorkspace*:
+
+* peak_position : the peak position (workspace index) used to define the :math:`\theta` angles (detector positions)
```

Comment:
`:math:'2\theta'`? And the unit of degrees.

Reply:
The unit is given below, for me it seems to be ok to say that there is an angle of name 2theta.
I review the naming in general.